person/src/com/person/utils/Validate.java

```diff
@@ -0,0 +1,21 @@
+package com.person.utils;
+
+public class Validate {
+    public static boolean name(String name, String rx)
```

Comment:
@tatevikb you can use it not only for name right ? so make a name generic

Reply:
@nsimonyan OK, I moved name -> names.